src/components/shared/NavBar.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="navbar">
+    <nav>
+      <ul>
+        <li><router-link to="/">Home</router-link></li>
+        <li class="sep">|</li>
+        <li><router-link to="/dining">Dining</router-link></li>
+        <li class="sep">|</li>
+        <li><router-link to="/events">Events</router-link></li>
+        <li class="sep">|</li>
+        <li><router-link to="/spa">Spa</router-link></li>
+      </ul>
+    </nav>
+  </div>
+</template>
+  
+<script>
+  export default {
+    name: 'NavBar'
+  }
+</script>
+  
+<style scoped>
+  .navbar {

```

Comment:
Most of this will go away, if you implement the simpler version for the NavBar. But in general, the css selectors should be nested when possible rather than all at the top level.